appstore/developer_portal_api.py

```diff
@@ -0,0 +1,629 @@
+
+            # Get developer ID from auth (This is also where we check the user is authenticated)
+            result = authed_request('GET', f"{config['REBBLE_AUTH_URL']}/api/v1/me/pebble/appstore")
+            if result.status_code != 200:
+                abort(401)
+            me = result.json()
+            developer_id = me['id']
+
+            # Find developer
+            developer = Developer.query.filter(Developer.id == developer_id).one_or_none()
+
+            if developer is None:
+                return jsonify(error = "You do not have an active developer account.", e = "account.invalid", message = "Please visit dev-portal.rebble.io to activate your developer account"), 409
+
+            # Upload banner if present
+            if "banner" in request.files:
+                header_asset = upload_asset(request.files["banner"], request.files["banner"].content_type)
+            else:
+                header_asset = None
+
+            # Copy screenshots to platform map
+            for platform in screenshots:
+                for x in range(6):
+                    if f"screenshot-{platform}-{x}" in request.files:
+                        screenshots[platform].append(request.files[f"screenshot-{platform}-{x}"])
+
+            # Remove any platforms with no screenshots
+            clearedScreenshots = dict(screenshots)
+            for platform in screenshots:
+                if len(clearedScreenshots[platform]) < 1:
+                    del clearedScreenshots[platform]
+            screenshots = clearedScreenshots
+
+            # Add blanks to optional values
+            for x in ["source","website"]:
+                if not x in params:
+                    params[x] = ""
+
+            app_obj = App(
+                id = id_generator.generate(),
+                app_uuid = appinfo['uuid'],
+                asset_collections = { x: AssetCollection(
+                    platform=x,
+                    description=params['description'],
+                    screenshots=[upload_asset(s, s.content_type) for s in screenshots[x]],
+                    headers = [header_asset] if header_asset else [],
+                    banner = None
+                ) for x in screenshots},
+                category_id = category_map[params['category']],
+                companions = {}, # companions not supported yet
+                created_at = datetime.datetime.utcnow(),
+                developer = developer,
+                hearts = 0,
+                releases = [],
+                icon_large = upload_asset(request.files['large_icon'], request.files["large_icon"].content_type),
+                icon_small = upload_asset(request.files['small_icon'], request.files["small_icon"].content_type) if 'small_icon' in params else '',
+                source = params['source'],
+                title = params['title'],
+                type = params['type'],
+                timeline_enabled = False,
```

Comment:
Why is this forced false?

Reply:
Currently that parameter is ignored by appstore_api everywhere. 
https://discord.com/channels/221364737269694464/407270816997441547/827655849325232170